api/adobe_vendor_id.py

```diff
@@ -147,6 +229,36 @@ def error_document(self, type, message):
             vendor_id=self.vendor_id, type=type, message=message)
 
 
+class DeviceManagementRequestHandler(object):
+    """Handle incoming requests for the DRM Device Management Protocol."""
+        
+    def __init__(self, credential):
+        self.credential = credential
+        
+    def device_list(self):
+        return "\n".join(
+            sorted(
+                x.device_identifier
+                for x in self.credential.drm_device_identifiers
+            )
+        )
+
+    def register_device(self, data):
+        device_ids = data.split("\n")
+        if len(device_ids) > 1:
+            return PAYLOAD_TOO_LARGE.detailed(
+                _("You may only register one device ID at a time.")
```

Comment:
I would say the spec is silent on this issue. I can add something to the spec saying that the implementing server may put limits on how many IDs can be registered at once.